### ISSUE TITLE
Plans to update the library to v5?

### ISSUE THREAD

**Issue by djzwerg** [backdrop-contrib/photoswipe#30]
See here: https://github.com/dimsemenov/PhotoSwipe/issues/1749

**Comment by korontari** [comment]
Yeah, it would be great to update the library, looks very promising. Though it is in beta yet and we should possibly wait until release.
